scrunch/helpers.py

```diff
@@ -109,12 +109,27 @@ def _validate_category_rules(categories, rules):
         )
 
 
-def shoji_entity_wrapper(payload):
+def shoji_entity_wrapper(body, **kwargs):
     """
     receives a dictionary and wraps its content on a body keyed dictionary
     with the appropiate shoji:<class> attribute
     """
-    return {
+    payload = {
         'element': 'shoji:entity',
-        'body': payload
+        'body': body
     }
+    payload.update(**kwargs)
+    return payload
+
+
+def shoji_catalog_wrapper(index, **kwargs):
```

Comment:
Is this dead code now?

Reply:
you're right, I'm not using it anymore. But I can see places where I'll need it in more tests. I don't want to pollute this PR with those changes though. It's _future code_